app/policies/user_policy.rb

```diff
@@ -34,4 +34,10 @@ def resolve
       scope
     end
   end
+
+  private
+
+  def draw_intent_state
+    !user.draw.intent_locked && !user.draw.draft? if user.draw
```

Comment:
that's what I had before and this is the problem I was running into yesterday -- the idea is that if admins want, they can allow students to change their intent in any phase. 

I can change this, but I figured more flexibility would be better?